app/views/users/mailer/password_change.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>Hello <%= @resource.email %>!</p>
```

Comment:
This should also be the username. Or we should remove it. I kind of like the idea of removing it because the less information we give a potential attacker, the better.